deps.ts

```diff
@@ -0,0 +1,5 @@
+export { Drash } from "https://deno.land/x/drash@v1.0.5/mod.ts";
+export {
+  ServerRequest,
+} from "https://deno.land/std@v0.56.0/http/server.ts";
+export { vary } from "https://raw.githubusercontent.com/dmpjs/vary/v1.0.3/mod.ts";
```

Comment:
Place vary in your middleware directory's deps.ts file. We're going to keep third-party modules separated from the root deps.ts file.